test/powershell/engine/Basic/DefaultConfirmImpact.Tests.ps1

```diff
@@ -0,0 +1,271 @@
+            @{ Cmdlet = 'Start-Process'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Start-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Start-Sleep'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Computer'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Job'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Process'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Suspend-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Tee-Object'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Connection'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Json'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-ModuleManifest'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Path'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-PSSessionConfigurationFile'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Trace-Command'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unblock-File'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unregister-Event'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unregister-PSSessionConfiguration'; ConfirmImpact = 'Low' }
+            @{ Cmdlet = 'Update-FormatData'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Update-Help'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Update-TypeData'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Debugger'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Event'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Job'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Process'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Where-Object'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Debug'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Error'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Host'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Information'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Output'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Progress'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Verbose'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Warning'; ConfirmImpact = 'Medium' }
+        ) # $DefaultCommands
+
+
+        $CommandList = Get-Command -ListAvailable -CommandType Cmdlet |
+            ForEach-Object {
+            $type = $_.ImplementingType
+
+            if ($type) {
+                $Attribute = $type.GetCustomAttributes($true) |
+                    Where-Object { $_.VerbName -ne $null }
+
+                '{0}-{1}' -f $Attribute.VerbName, $Attribute.NounName
+            }
+        }
+    }
+
+    It '<Cmdlet> should have a ConfirmImpact rating of <ConfirmImpact>' -TestCases $DefaultCommands {
+        param($Cmdlet, $ConfirmImpact)
+
+        $Cmdlet | Should -BeIn $CommandList
```

Comment:
I think we can remove the check - it is not related the PR.

Reply:
I was hoping to somehow verify that all default commands are being properly tested in this fashion... but it doesn't seem to like what I'm trying to do at the moment. 😄